semantic/test/Semantic/CLI/Spec.hs

```diff
@@ -34,31 +33,31 @@ testTree = testGroup "Semantic.CLI"
 renderDiff :: String -> String -> [String]
 renderDiff ref new = unsafePerformIO $ do
   let check p = do
-        exists <- Path.doesFileExist (Path.absRel p)
+        exists <- doesFileExist p
         unless exists (throwIO (userError ("Can't find path " <> p)))
 
   check ref
   check new
-  useJD <- (Path.hasExtension ".json" (Path.absRel ref) &&) <$> fmap isJust (Path.findExecutable "jd")
+  useJD <- (takeExtension ref == ".json" &&) <$> fmap isJust (findExecutable "jd")
```

Comment:
`pathtype` generally mimics `filepath`'s API pretty closely, so most of the changes either just drop conversions to/from strings, or drop the module prefix and use the mimicked symbol. In this case, `pathtype` does something completely different for some reason, so there's some actual logic to this change.

I did double-check that `takeExtension` returns the `.`.